css/at-rules.json

```diff
@@ -123,7 +133,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@document"
   },
   "@font-palette-values": {
-    "syntax": "@font-palette-values <dashed-ident> { <declaration-list> }",
+    "syntax": "@font-palette-values <dashed-ident> {\n  <declaration-list>\n}",
```

Comment:
Is this intentional?

Reply:
Yes, just a formatting adjustment to keep it consistent with other at-rules